packages/components/.storybook/components/DecorativeContent.tsx

```diff
@@ -0,0 +1,23 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
```

Comment:
can you add a comment that makes it apparent that this renders a checker pattern?